### PR TITLE
[LS] Add checker support for `BlockchainHelpers` identifier location

### PR DESCRIPTION
**Pull request by m-Peter**
**Depends on:** https://github.com/onflow/cadence-tools/pull/181

## Description

The Cadence testing framework has introduced a new identifier location: `BlockchainHelpers`, so we add support for it in the language server. Otherwise the checking would fail.

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence-lint/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: languageserver/go.mod [hunk 1]
```diff
@@ -177,3 +186,7 @@ require (
 	modernc.org/memory v1.5.0 // indirect
 	modernc.org/sqlite v1.21.1 // indirect
 )
+
+replace github.com/onflow/cadence-tools/test => ../test
+
+replace github.com/onflow/cadence => ../../cadence
```
Comment (turbolent) [inline languageserver/go.mod:192]: `replace` statements should not be comitted

```suggestion
```

**Review by turbolent — COMMENTED** [review]
👍

File: languageserver/go.mod (hunk 1 above)
Comment (SupunS) [reply languageserver/go.mod:192]: maybe refer to the commit, until we tag a release

Comment (m-Peter) [reply languageserver/go.mod:192]: I think I have tried that before, but it didn't work with forks. I will check again though :+1:

**Review by SupunS — APPROVED** [review]

**Pull request merged**